main.js

```diff
@@ -1,6 +1,59 @@
 const addButton = document.getElementById('add_button');
 const booksWrapper = document.getElementById('book-collection-wrapper');
 let populateBookSection = () => {};
+const navBtns = document.querySelectorAll('#menu li');
+
+// NAVIGATION MENU LOGIC\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\
+// onclick receives the index of which btn was clicked
+function HighlightSelection(index) {
+  navBtns.forEach((btn, i) => {
+    btn.classList.remove('selected');
+    if (index === i) {
+      // checks which button was clicked and adds a class
+      btn.classList.add('selected');
+    }
+  });
+}
+
+function navigate(index) {
+  const sections = ['list', 'add-new', 'contact'];
+  const bookCollection = document.getElementById('book-collection');
+  const addBookSection = document.getElementById('add-book-section');
+  const contactSection = document.getElementById('contact-section');
+  switch (sections[index]) {
+    // compares and changes the elements class accordingly to show or hide the sections
+    case 'list': {
+      bookCollection.classList.add('show');
+      addBookSection.classList.remove('show');
+      contactSection.classList.remove('show');
+      HighlightSelection(index);
+      break;
+    }
+    case 'add-new': {
+      bookCollection.classList.remove('show');
+      addBookSection.classList.add('show');
+      contactSection.classList.remove('show');
+      HighlightSelection(index);
+      break;
+    }
+    case 'contact': {
+      bookCollection.classList.remove('show');
+      addBookSection.classList.remove('show');
+      contactSection.classList.add('show');
+      HighlightSelection(index);
+      break;
+    }
+    default: {
+      break;
+    }
+  }
+}
+
+// loop through btns adding onclick
+navBtns.forEach((btn, index) => {
+  btn.onclick = () => navigate(index);
+});
+// BOOK COLLECTION CLASS\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\
 class Books {
```

Comment:
- [ ] As the previous reviewer mentioned the book can't be added to the array and it produces an error on the console `Uncaught TypeError: Cannot read properties of null (reading 'length')`, kindly make sure to check this out

![Screenshot_20230811_170654](https://github.com/Estete9/awesomeBooks/assets/25227022/0de7be0e-375e-4a05-8d81-1fae4a744eb1)

Reply:
Kindly fix the error suggested by the previous reviewer, currently, a book can't be added or removed and this is a major requirement for the project. Please fix this error and the error on the console so that you'd have a good working app

![image](https://github.com/Estete9/awesomeBooks/assets/20537712/90a5dbba-9104-4edf-b4cd-146de9ca1ed6)